tests/generation/test_hub.py

```diff
@@ -25,29 +27,45 @@
 
 @is_inferentia_test
 @requires_neuronx
-def test_model_from_hub():
-    model = NeuronModelForCausalLM.from_pretrained(
-        "dacorvo/tiny-random-gpt2-neuronx", revision="1b3456cf877cc42c053ee8464f1067021eccde4b"
-    )
+@pytest.mark.parametrize(
+    "model_id, revision",
+    [
+        ["dacorvo/tiny-random-gpt2-neuronx", "1b3456cf877cc42c053ee8464f1067021eccde4b"],
+        ["dacorvo/tiny-random-gpt2-neuronx-no-checkpoint", "78eb2313ab7e149bbc22ff32257db93ba09e3033"],
```

Comment:
For the test with no checkpoint, shall we check there is no checkpoint after push to hub?

Reply:
It is a download here: the check is done in the other test.